redis_collections/dicts.py

```diff
@@ -75,43 +64,55 @@ def __init__(self, *args, **kwargs):
             As mentioned, :class:`Dict` does not support following
             initialization syntax: ``d = Dict(a=1, b=2)``
         """
+        data = args[0] if args else kwargs.pop('data', None)
```

Comment:
I don't love this, but my new `update` method needs some special things and I didn't want to break `List` and `Set`.